go.sum

```diff
@@ -1,9 +1,9 @@
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
-github.com/astarte-platform/astarte-go v0.0.0-20210219104629-4a237278377b h1:zdeYDwlOGD4IysbnmGxvqzgW90X37oy1EzPR1UV05EY=
-github.com/astarte-platform/astarte-go v0.0.0-20210219104629-4a237278377b/go.mod h1:mW8KuuYUMvO91z4HHYpa3e841jZPnp269fuXGl+c8WU=
+github.com/astarte-platform/astarte-go v0.90.0 h1:u6n1KslO0inOfHA37f5aRkcdhhTKKDgOWvlR6F2euSY=
+github.com/astarte-platform/astarte-go v0.90.0/go.mod h1:z3qfyKDldol0qjw6ongq72DmMj2TbwNzxLYhcVm+tF4=
 github.com/aws/aws-sdk-go v1.34.28/go.mod h1:H7NKnBqNVzoTJpGfLrQkkD+ytBA93eiDYi/+8rV9s48=
-github.com/cristalhq/jwt/v3 v3.0.11 h1:oQAo2wlS8O/BUG03yIlDRzBrCwdAOiP52M1QRCk7MzI=
```

Comment:
Something went wrong here due to a wrong tagging/release of astarte-go. Please release astarte-go 0.9.1, update the dependency, then it's good to go.